### PR TITLE
Removed download icon when ISO is not extractable for users

### PR DESCRIPTION
**Pull request by Spaceman1984**
### Description

When an ISO is not marked as extractable, the download button should not be displayed in the UI.

Fixes: #4535 

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### How Has This Been Tested?

This was tested by uploading an iso as a user with and without the extractable switch enabled.
Tested extraction of ISO by calling the API and by using the UI.
Tested extraction by Admin.
Tested extraction by a user in a project.

### CODE REVIEW

**Comment by Spaceman1984** [comment]
@blueorangutan package

File: ui/src/config/section/image.js [hunk 1]
```diff
@@ -268,7 +268,7 @@ export default {
           dataView: true,
           show: (record, store) => {
             return (['Admin'].includes(store.userInfo.roletype) || // If admin or owner or belongs to current project
-              (record.domainid === store.userInfo.domainid && record.account === store.userInfo.account) ||
+              (record.domainid === store.userInfo.domainid && record.account === store.userInfo.account && record.isextractable) ||
               (record.domainid === store.userInfo.domainid && record.projectid && store.project && store.project.id && record.projectid === store.project.id)) &&
               !(record.account === 'system' && record.domainid === 1) &&
               record.isready
```
Comment (shwstppr) [inline ui/src/config/section/image.js:274]: @Spaceman1984 shouldn't `isextractable` condition be added with `isready` irrespective of same account or not?

Comment (Spaceman1984) [reply ui/src/config/section/image.js:274]: No - admins can download all ISOs

**Comment by shwstppr** [comment]
@blueorangutan ui

**Review by utchoang — APPROVED** [review]
LGTM

**Review by rhtyd — APPROVED** [review]

**Pull request merged**